### PR TITLE
Added command to toggle off the display of the full stack price

### PR DESCRIPTION
**Pull request by Tatwi**

### CODE REVIEW

File: BetterVendorPrice/BetterVendorPrice.lua [hunk 1]
```diff
@@ -87,6 +89,13 @@ function BVP.Slash(arg) -- can't be a : because used directly as slash command
       BVP:SetSaved("debug", tonumber(rest))
     end
     BVP:PrintDefault("BetterVendorPrice debug now %", BVP.debug)
+  elseif BVP:StartsWith(arg, "fs") then
```
Comment (mooreatv) [inline BetterVendorPrice/BetterVendorPrice.lua:92]: First thanks a lot for taking the time to make a PR !

you can use cmd == "f" as nothing else starts with f so far

also usually for command changing flags it's nice to have on/off/toggles not just the toggle

but this being said, for this addon, why not add a boolean checkbox in the options menu? if you want maybe I can do that quickly

**Comment by mooreatv** [comment]
Also can you elaborate on your need in this case, do you wish the output to be more compact ?

if we remove 1 line for full stack price, we should put the stacking information somewhere else on the other lines, no ?

**Comment by Tatwi** [comment]
Really it was just a matter of personal preference, but I figured I may as well submit it for consideration as it might be handy. I'm not actually familiar with the WoW addons way of doing things, I just know Lua from working on other projects. 

Personally, I set it up so that "full stack price" is off by default and the tooltips read like so,

**Small Spider Leg**
Each: 3c
This Stack: 6c

basically because I am _THAT_ anal retentive lol... 

Great mod, love it! Thanks for making it! :)

**Comment by mooreatv** [comment]
So I made a config option and changed the code to try to still carry the stack size in a compact format:

with that option:
image

compared to the default/current layout:
image

I have an english as a second language question though, which is: should I put
(per; stacks in X)
oe
(per; stacks by X)

I don't want to put (stacks of X) as that would imply the price is for the stack; thus the verb form

suggestions?

**Comment by mooreatv** [comment]
shipped in v1.05 let me know what you think?

**Comment by Tatwi** [comment]
Sorry, I was away...

I think you did nice work, thanks! As far as your English goes, it's totally fine. However, I found that there were too many words being used in the tooltips, which ended up being distracting. So on my client I changed them to look like this, 
tooltip
because that was really all the information I personally want/need to see. I think that amount of wording would be sufficient for any native English speaker as well.

Honestly though, there is nothing _wrong_ with what you have created - thanks again for making it!

**Pull request closed without merging**